src/connectWithActions.js

```diff
@@ -4,7 +4,7 @@ import { connect } from 'react-redux';
 
 import type { Dispatch, MapStateToProps } from './types';
 import * as actions from './actions';
-import { isStateGoingBack } from './utils/misc';
+import { areStatesSeemsToBeEqual } from './utils/misc';
```

Comment:
Correct grammatically `doStatesSeemsToBeEqual`.
Still terrible name ;)

Reply:
Done 👍 :)

Reply:
Actually this is ungrammatical too ;) -- `doStatesSeemToBeEqual` would fix the grammar.